central/cve/datastore/datastore.go

```diff
@@ -34,14 +36,15 @@ type DataStore interface {
 }
 
 // New returns a new instance of a DataStore.
-func New(graphProvider graph.Provider, storage store.Store, indexer index.Indexer, searcher search.Searcher) (DataStore, error) {
+func New(graphProvider graph.Provider, indexQ queue.WaitableQueue, storage store.Store, indexer index.Indexer, searcher search.Searcher) (DataStore, error) {
```

Comment:
Dackbox does lazy indexing. In cases where we want the indexing to happen actively so that the UI reflects the change, we trigger it. Nonetheless, that store is going away soon, and we won't be dealing with it.